website/docs/cli/commands/fmt.mdx

```diff
@@ -55,6 +55,6 @@ The command-line flags are all optional. The following flags are available:
 * `-list=false` - Don't list the files containing formatting inconsistencies.
 * `-write=false` - Don't overwrite the input files. (This is implied by `-check` or when the input is STDIN.)
 * `-diff` - Display diffs of formatting changes
-* `-check` - Check if the input is formatted. Exit status will be 0 if
+* `-check` - Check if the input is formatted. Displays the filenames that need formatting correction. Exit status will be 0 if
   all input is properly formatted and non-zero otherwise.
```

Comment:
Perhaps something like this, to make it clear that there is no output in the success case.

```suggestion
* `-check` - Check if the input is formatted. Exit status will be 0 if all input is properly formatted. If not, exit status will be non-zero and the command will output a list of filenames whose files are not properly formatted.
```